src/script/auth/page/ConversationJoin.tsx

```diff
@@ -124,10 +124,12 @@ const ConversationJoin = ({
     }
   }, []);
 
-  const routeToApp = () => {
+  const routeToApp = (conversation?: string, domain?: string) => {
     const redirectLocation = isPwaEnabled
       ? UrlUtil.pathWithParams(EXTERNAL_ROUTE.PWA_LOGIN, {[QUERY_KEY.IMMEDIATE_LOGIN]: 'true'})
-      : UrlUtil.pathWithParams(EXTERNAL_ROUTE.WEBAPP);
+      : `${UrlUtil.pathWithParams(EXTERNAL_ROUTE.WEBAPP)}${
+          conversation && `#/conversation/${conversation}${domain && `/${domain}`}`
```

Comment:
In this case `${domain && `/${domain}`}`, if domain === undefined you get on the end of the link `#/conversation/<conversation>undefined`, please change to `${domain ? `/${domain}` : ''}` to avoid undefined on the end :)

Reply:
good  catch :+1: